documenting.rst

```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
+docs.python.org. There are several documentation translations already
+in production, other are work in progress:
+
++--------------+--------------------+------------------------------------------+
+| Language     | Contact            | Links                                    |
++==============+====================+==========================================+
+| Arabic (ar)  | Abdur-Rahmaan      | `github <https://github.com/             |
+|              | Janhangeer         | Abdur-rahmaanJ/py-docs-ar>`__            |
++--------------+--------------------+------------------------------------------+
+| Bengali as   | `Kushal Das        | `github <https://github.com/python/      |
+| spoken in    | <http://bugs.python| python-docs-bn-in>`__                    |
+| India (bn_IN)| .org/user16382>`__ |                                          |
++--------------+--------------------+------------------------------------------+
+| French (fr)  | `Julien Palard     | `github <https://github.com/python/      |
+|              | (mdk) <https://bug | python-docs-fr/>`__                      |
+|              | s.python.org/user2 | `doc <https://docs.python.org/fr/>`__    |
+|              | 3063>`__           |                                          |
++--------------+--------------------+------------------------------------------+
+| Hindi as     |                    | `github <https://github.com/             |
+| spoken in    |                    | CuriousLearner/python-docs-hi-in>`__     |
+| India        |                    |                                          |
+| (hi_IN)      |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Hungarian    | `Tamás Bajusz      | `github <https://github.com/python/      |
+| (hu)         | (gbtami) <http://b | python-docs-hu/>`__                      |
+|              | ugs.python.org/use |                                          |
+|              | r25857>`__         | `list <https://mail.python.org/pipermail/|
+|              |                    | python-hu/>`__                           |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Italian (it) |                    | `mail <https://mail.python.org/pipermail/|
+|              |                    | doc-sig/2019-April/004114.html>`__       |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Japanese     | `Inada Naoki       | `github <https://github.com/python/      |
+| (ja)         | (methane) <https:/ | python-docs-ja/>`__                      |
+|              | /bugs.python.org/u |                                          |
+|              | ser10371>`__       | `doc <https://docs.python.org/ja/>`__    |
```

Comment:
Please change contact to @cocoatomo https://bugs.python.org/user19001

Reply:
OK done.